scipy/stats/_discrete_distns.py

```diff
@@ -152,6 +152,79 @@ def _entropy(self, p):
 bernoulli = bernoulli_gen(b=1, name='bernoulli')
 
 
+class betabinom_gen(rv_discrete):
+    r"""A beta-binomial discrete random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability mass function for `betabinom` is:
+
+    .. math::
+
+       f(k) = \binom{n}{k} \frac{B(k + a, n - k + b)}{B(a, b)}
+
+    for ``k`` in ``{0, 1,..., n}``, where :math:`B(a, b)` is the beta function.
+
```

Comment:
Could you add a short explanation that it is a binomial distribution with random success probability p that follows a beta distribution?

Reply:
Done.